src/Block.cpp

```diff
@@ -0,0 +1,73 @@
+#include "Block.hpp"
+
+#include "Declarations.hpp"
+#include "FreeBlock.hpp"
+
+/*
+ * Takes 16 bytes in memory
+ * 1 byte for the used flag
+ * 8 bytes for the typeDescriptor pointer
+ * 7 bytes padding
```

Comment:
since we do not use the 'used'-bit anymore

```suggestion
 * Takes 8 bytes in memory for the typeDescriptor pointer
```